### PR TITLE
[TSC] Adding the artefacts

### PR DESCRIPTION
**Pull request by prabhubalan**
Adding the diagram in the artefacts for future change

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@prabhubalan can you please create the table in Markdown? you can use this utility : http://www.tablesgenerator.com/markdown_tables

**Comment by prabhubalan** [comment]
> @prabhubalan can you please create the table in Markdown as per @CsatariGergely input? you can use this utility : http://www.tablesgenerator.com/markdown_tables

Sure @rabi-abdel . I will create the table as md format

**Comment by prabhubalan** [comment]
> > @prabhubalan can you please create the table in Markdown as per @CsatariGergely input? you can use this utility : http://www.tablesgenerator.com/markdown_tables
> 
> Sure @rabi-abdel . I will create the table as md format

@rabi-abdel - I have added the table in the form of MarkDown

File: doc/tech/release_notes/release_process.md [hunk 1]
```diff
@@ -30,7 +30,30 @@ During the development cycle when working with release branches, developers or a
...
+| Release Candidate | ||~~13/Apr/2020~~| 11/May/2020 |
+| Baldy Sign-Off | ||~~17/Apr/2020~~| 15/May/2020 |
+| **Baraque (Proposed)** |
+| M1 (Release Planning/Requirements) | ||| 29/May/2020 | ~~17/Jun/2020~~ |
+| M2 (Issue Logging) | |||| 19/Jun/2020 | ~~01/Jul/2020~~ |
+| M3 (Freeze Contributions) | | | | | | | | 14/Sep/2020 |
+| M4 (Freeze Proof Reading) | | | | | | | | 18/Sep/2020 |
+| Release Candidate | | | | | | | | 21/Sep/2020 |
+| Baldy Sign-Off | | | | | | | | 25/Sep/2020 |
+
```
Comment (kedmison) [inline doc/tech/release_notes/release_process.md:56]: This is really good.  
In terms of horizontal white-space, though, I think we can optimize it a bit by removing the year from most cells, as the year is already in the column header.  
```suggestion
| Title | Feb/20 | Mar/20 | Apr/20 | May/20 | Jun/20 | Jul/20 | Aug/20 | Sep/20 | 
|------|-------|-------|--------|---------|-------|-------|--------|---------|
| Events | 
| ONES NA 2020 ||| 20/Apr to 23/Apr |
| ONES - Antwerp, Belgium (Proposed Release Name: **Baraque**) | | | | | | | | 29/Sep to 2/Oct | 
| Long Holidays |
| Easter (UK/US) |||10/Apr to 13/Apr | 
| Labour Day (China) | ||| 1/May to 5/May |
| Dragon Boat Festival (China) ||||| 25/Jun to 27/Jun |
| **Baldy (Proposed)** |
| M1 (Release Planning/Requirements) | 1/Feb |
| M2 (Issue Logging) | 21/Feb |
| M3 (Freeze Contributions) | ||~~03/Apr~~| 01/May |
| M4 (Freeze Proof Reading) | ||~~09/Apr~~| 07/May |
| Release Candidate | ||~~13/Apr~~| 11/May |
| Baldy Sign-Off | ||~~17/Apr~~| 15/May |
| **Baraque (Proposed)** |
| M1 (Release Planning/Requirements) | ||| 29/May | ~~17/Jun~~ |
| M2 (Issue Logging) | |||| 19/Jun | ~~01/Jul~~ |
| M3 (Freeze Contributions) | | | | | | | | 14/Sep |
| M4 (Freeze Proof Reading) | | | | | | | | 18/Sep |
| Release Candidate | | | | | | | | 21/Sep |
| Baldy Sign-Off | | | | | | | | 25/Sep |

```

Comment (prabhubalan) [reply doc/tech/release_notes/release_process.md]: > This is really good.
> In terms of horizontal white-space, though, I think we can optimize it a bit by removing the year from most cells, as the year is already in the column header.

Thanks @kedmison. Removed the year in each cells. Now the data fits into the table without horizontal scroll bar.

**Comment by rabi-abdel** [comment]
@kedmison @CsatariGergely please re-review/approve

**Pull request merged**